examples/movie_view_ratings.py

```diff
@@ -206,6 +183,9 @@ def compute_on_local():
 
 
 def main(unused_argv):
+
+    delete_if_exists(FLAGS.output_file)
```

Comment:
delete_if_exists is needed only for spark, and with current code it will fail for Beam
Nit: move it to `if spark` clause